README.md

```diff
@@ -0,0 +1,330 @@
+# Billpay Module
+
+## Installation
+
+1 Add to composer.json 
```

Comment:
Please use the core approach of composer require for billpay eco module and its lib if possible (should be via packagist).
Espcecially "url": "git@github.com:spryker-eco/billpay.git" should never be needed.